runtime/src/main/java/io/quarkiverse/cxf/transport/VertxHttpServletRequest.java

```diff
@@ -449,7 +449,9 @@ public String getRequestURI() {
 
     @Override
     public StringBuffer getRequestURL() {
-        return new StringBuffer(request.absoluteURI());
+        String absoluteUri = request.absoluteURI();
+        String urlWithoutParams = absoluteUri.replaceFirst("\\?.*$", "");
```

Comment:
Do we know how often this method is called per "typical" request?
I'm asking because here a thrown-away `Pattern` is created under the hood, so that each invocation will go through a re-compilation the `Pattern`.
I think the `Pattern` should be pre-compiled once (as a constant).

Reply:
Nice catch Falko!  Each request invocation will have a call to this method, so you're quite right, there will be a performance hit here.  I think we can do a "0.13.1" release with your logging updates and this change?

Reply:
Sounds good! I'll have a look at the logging stuff tomorrow.

Reply:
Sounds good, thanks.  New PR submitted:  https://github.com/quarkiverse/quarkus-cxf/pull/351